### PR TITLE
Build with -D LLVM_ENABLE_LIBCXX

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,7 +36,6 @@ source:
 build:
   number: {{ build_number }}
   string: {{ llvm_variant }}_{{ build_number }}
-  skip:  true  # [py<36]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Was this intentional? Seems like this would end up with old VCs showing up.

**Comment by jakirkham** [comment]
Is there some file we can test for?

File: recipe/meta.yaml (hunk 1 above)
Comment (SylvainCorlay) [inline recipe/meta.yaml]: no, I need to fix it.

**Comment by SylvainCorlay** [comment]
I am not sure that this is the way to go.

I guess that if someone wants to embed llvm in an application that is not built with clang, they will want to use the gcc runtime.... Not sure in fact.

**Pull request closed without merging**